daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy-without-cli.md

```diff
@@ -0,0 +1,299 @@
+---
+type: docs
+title: "Deploy Dapr at a Kubernetes cluster"
+linkTitle: "Deploy Dapr without CLI"
+weight: 21000
+description: "Follow these steps to deploy Dapr at Kubernetes without cluster permissions."
+aliases:
+    - /getting-started/install-dapr-kubernetes/
+---
+
+This part describes the process of installation Dapr without CLI or Helm when you have no cluster permissions. Such situation may arise when you have a project in OKD or namespace in Kubernetes. In this case you have no cluster permission and can't deploy the operator. You can't also address cluster admin to install Dapr operator because you are the only one who needs it.
+
+This approach can also be used with other orchestrators, for example in Docker Swarm.
+
+## Prerequisites
+
+- Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Kubernetes cluster (see below if needed)
+
+### Create cluster
+
+You can install Dapr at any Kubernetes cluster. Here are some helpful links:
+
+- [Setup Minikube Cluster]({{< ref setup-minikube.md >}})
+- [Setup Azure Kubernetes Service Cluster]({{< ref setup-aks.md >}})
+- [Setup Google Cloud Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart)
+- [Setup Amazon Elastic Kubernetes Service](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+
+
+## Install without Dapr CLI
+
+You can install Dapr to a Kubernetes cluster or other orchectrator without Dapr CLI.
+
+### Install Dapr
+
+Dapr installation is done with ordinary Kubernes deployment. 
```

Comment:
Typo Kubernes should be Kubernetes. 

I suggest this sentence is reworded to  "Dapr is installed as a Kubernetes deployment."